Controller/Admin/ModuleSupport.php

```diff
@@ -203,18 +205,20 @@ private function _validateRequest()
         $sFromEmail = $this->getConfig()->getRequestParameter('module_support_email_from');
         $sReplyToEmail = $this->getConfig()->getRequestParameter('module_support_email_reply');
 
-        $this->_aViewData['replyToEmail'] = $sReplyToEmail;
-        $this->_aViewData['fromEmail'] = $sFromEmail;
-        $this->_aViewData['body'] = $sBody;
+        Helper::addToViewData($this, [
+            'replyToEmail' => $sReplyToEmail,
+            'fromEmail' => $sFromEmail,
+            'body' => $sBody,
+        ]);
 
         // there are two separate validation methods because $sFromEmail is mandatory and $sReplyToEmail
         // is optional - it only needs to be validated if it was set
         if (!$this->_isFromEmailValid($sFromEmail) || !$this->_isReplyToEmailValid($sReplyToEmail)) {
-            throw new Exception(Helper::translate('wd_enter_valid_email_error'));
+            throw new StandardException(Helper::translate('wd_enter_valid_email_error'));
```

Comment:
Just a quick question regarding the translation within this exception. Why do we have a translated string if the users don't see the message? Or did that point regarding the exceptions not being shown to users only at that point? (reference [review ](https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMTcyNDk4OTY0OnYy/pull_request_review_threads/discussion_))

Reply:
Exceptions from this method (`_validateRequest`) are caught and shown to the user in the `sendSupportEmailAction` method. This is not the case in the example you pointed to.